src/main/java/seedu/address/logic/commands/DarkModeCommand.java

```diff
@@ -0,0 +1,20 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import seedu.address.model.Model;
+
+/**
+ * Changes the style of NeoBook to dark mode.
+ */
+public class DarkModeCommand extends Command {
+    public static final String COMMAND_WORD = "dark";
+
+    public static final String MESSAGE_SUCCESS = "Changed NeoBook to dark mode.";
+
+    @Override
+    public CommandResult execute(Model model) {
+        requireNonNull(model);
+        return new CommandResult(MESSAGE_SUCCESS, false, false, false, true);
+    }
```

Comment:
Perhaps there could be a check for whether the app is already in dark mode since there is access to the `model`, in which case `Theme.NO_CHANGE` would be passed and a more relevant message would be shown to the user. (similarly for `LightModeCommand`)